static/css/main.css

```diff
@@ -7,68 +7,127 @@ body {
   font-size: 16px;
   font-family: 'Lato', sans-serif; }
 
-.container {
-  width: 100%; }
+.encloser {
+  display: flex;
+  min-height: 100vh;
+  flex-direction: row; }
+
+@media (max-width: 768px) {
+  .encloser {
+    flex-direction: column; } }
 
 ul {
   list-style: none; }
 
-.alert-container {
-  position: fixed;
-  top: 0;
-  width: 100%;
-  z-index: 10; }
-
-.message-box ul li {
-  display: block;
-  border: solid 1px #eee;
-  border-radius: 10px;
-  padding: 5px 10px;
-  margin-bottom: 5px;
-  width: 90%;
-  position: relative;
-  word-wrap: break-word; }
-  .message-box ul li .username {
-    display: block;
+header {
+  display: flex;
+  background-color: #1e202f;
+  color: #fff;
+  padding: 10px 10px 5px;
+  flex-direction: row;
+  justify-content: space-between; }
+  header .users-list {
+    order: -1; }
+    header .users-list .users-icon {
+      cursor: pointer; }
+    header .users-list ul {
+      display: none; }
+  header .logo {
+    text-align: center;
+    font-size: 24px;
     font-weight: bold; }
-  .message-box ul li.chat-outgoing {
-    float: right;
-    color: white;
-    background-color: #827cd2; }
-    .message-box ul li.chat-outgoing a {
-      text-decoration: underline;
-      color: white; }
-    .message-box ul li.chat-outgoing a:hover {
-      color: #ddd; }
-  .message-box ul li.chat-incoming {
-    float: left;
-    color: #333; }
+  header .menu-list {
+    display: none; }
+    header .menu-list i.fa {
+      margin-right: 5px; }
 
-.message-box ul li:hover .delete-message {
-  visibility: visible; }
+@media (min-width: 768px) {
+  header {
+    flex: 1;
+    min-width: 300px;
+    flex-direction: column;
+    justify-content: start;
+    height: 100vh; }
+    header .logo {
+      text-align: left;
+      order: -3; }
+    header .settings {
+      order: -2; } }
 
-.message-form {
-  position: fixed;
-  width: 100%;
-  left: 0px;
-  bottom: 0px;
-  padding: 5px;
-  background-color: white;
-  border-top: solid 1px #ccc; }
-  .message-form textarea {
-    /* this font-size disables zooming
-    on mobile safari, which breaks layout */
-    font-size: 16px;
-    width: calc(100% - 40px); }
-  .message-form button {
-    border: none;
-    padding: 5px;
-    float: right;
-    height: 54px;
-    vertical-align: bottom; }
-    .message-form button i {
-      color: #1e202f; }
+main {
+  display: flex;
+  flex-grow: 1; }
+  main .content {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1; }
+    main .content .message-box {
+      height: calc(100vh - 120px);
+      overflow-y: scroll;
+      padding-top: 5px; }
+      main .content .message-box ul li {
+        display: block;
+        border: solid 1px #eee;
+        border-radius: 10px;
+        padding: 5px 10px;
+        margin-bottom: 5px;
+        width: 90%;
+        position: relative;
```

Comment:
If this is in reference to `position: relative`, I believe it was originally used to position the "delete message" button absolutely within the message element itself. That element was removed at some point, maybe in the transition from backchannel, so the answer here is: nothing. :)